lib/temporal/client/grpc_client.rb

```diff
@@ -379,7 +379,7 @@ def cancel_polling_request
       def client
         @client ||= Temporal::Api::WorkflowService::V1::WorkflowService::Stub.new(
           url,
-          :this_channel_is_insecure,
+          Temporal.configuration.grpc_ssl_config,
```

Comment:
Would you mind passing this as the argument (similar to host/port) to the client to avoid this extra dependency?

Reply:
Still pull from config but do so here yeah?
 https://github.com/coinbase/temporal-ruby/blob/a7665ee115610f5eb6004d3dc92fbd297ef0e300/lib/temporal/client.rb#L18